client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React from "react";
+import { NavLink } from "react-router-dom";
 
-const MovieCard = props => {
-  return;
+const MovieCard = ({ movie }) => {
+  const { title, director, metascore, stars, id } = movie;
```

Comment:
Cool example of object destructuring, making the code cleaner and more readable.